### PR TITLE
ensure async functions and correct array handling in Profile component

### PR DESCRIPTION
**Pull request by ngduyd**

### CODE REVIEW

**Comment by ngduyd** [comment]
Refactor code lại giup với Minh, vscode t setting khác, sửa hết mấy cái dấu

File: src/app/(main)/liveButton.js
Comment (mimhle) [inline src/app/(main)/liveButton.js:1]: Rename file

File: src/app/(main)/profile/[id]/Profile.js [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+import Gallery from '@/app/(main)/Gallery';
+import Stat from './Stat';
+import Avatar from '../../Avatar';
+import CoverPhoto from '@/app/(main)/CoverPhoto';
+import LiveButton from '@/app/(main)/liveButton';
+import {
+    getUserData,
+    getLiveRooms,
+    setUserData,
+} from '../../../../../lib/frontend/api';
```
Comment (mimhle) [inline src/app/(main)/profile/[id]/Profile.js]: Refactor imports

File: src/app/(main)/actionButton.js
Comment (mimhle) [inline src/app/(main)/actionButton.js:1]: Rename to capitalize filename

File: src/app/(main)/List.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+'use client';
+
+import React, { useEffect, useState } from 'react';
+import { getUserData } from '../../../lib/frontend/api';
```
Comment (mimhle) [inline src/app/(main)/List.js]: Change to absolute path

File: src/app/(main)/profile/[id]/Stat.js [hunk 3]
```diff
@@ -1,46 +1,83 @@
-export default function Stat({ followers = 0, following = 0 }) {
+import { useState } from 'react';
+import ListComponent from '../../List';
```
Comment (mimhle) [inline src/app/(main)/profile/[id]/Stat.js]: See above

File: src/app/(main)/List.js
Comment (mimhle) [inline src/app/(main)/List.js:1]: List of what

**Pull request merged**